lib/active_node/validations.rb

```diff
@@ -34,6 +34,18 @@ module Validations
     autoload :UniquenessValidator, 'active_node/validations/uniqueness'
 
     module ClassMethods
+      # Initializes and saves an object in a single step.
+      def create(attributes = nil, &block)
+        if attributes.is_a?(Array)
+          attributes.collect { |attr| create(attr, &block) }
+        else
+          object = new(attributes)
+          yield(object) if block_given?
+          object.save
+          object
+        end
+      end
+
```

Comment:
Nice! What do I need to do to get it? BTW, I'll resume my work on this these days. I do have a couple more suggestions to make this more like ActiveRecord, and will include them as well.